app/controllers/rx_controller.rb

```diff
@@ -4,13 +4,21 @@
 class RxController < ApplicationController
   include ActionController::Serialization
 
-  skip_before_action :authenticate
+  before_action :authorize_rx
   before_action :authenticate_client
 
   protected
 
   def client
-    @client ||= Rx::Client.new(session: { user_id: ENV['MHV_USER_ID'] })
+    @client ||= Rx::Client.new(session: { user_id: mhv_correlation_id })
+  end
+
+  def authorize_rx
+    mhv_correlation_id || raise(Common::Exceptions::Unauthorized)
```

Comment:
403 seems appropriate to me, since the user is authenticated but not authorized.

It looks like I'm going to be working on an authorization system that can be used across the app, and I'll write up a design doc for that including information about our usage of status codes.